hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/KafkaOffsetGen.java

```diff
@@ -297,7 +295,37 @@ public String getTopicName() {
     return topicName;
   }
 
-  public HashMap<String, Object> getKafkaParams() {
+  public Map<String, Object> getKafkaParams() {
+    return kafkaParams;
+  }
+
+  private static Map<String, Object> excludeHoodieConfigs(TypedProperties props) {
```

Comment:
remove static ?

Reply:
Are you proposing to create `KafkaOffsetGen` object in `DeltaSync` as well to call `commitOffsetToKafka` method ? I wanted to avoid doing it as we are already doing so in respective Source classes. Also, I saw `excludeHoodieConfigs `  as helper method, hence created static

Reply:
@leesf done